cmd/k8s-target-retriever/main.go

```diff
@@ -44,15 +44,16 @@ func main() {
 
 	logrus.Infoln("connected to cluster, watching for targets")
 
-	for range time.Tick(time.Second * 5) {
+	for range time.Tick(time.Second * 7) {
```

Comment:
why 7 and not 5? should it be made configurable?

Reply:
this is in the "test" main, it is not used by the real pomi

Reply:
it is used to test an debug the targets retrieved